components/tools/OmeroWeb/omeroweb/webgateway/static/webgateway/js/ome.popup.js

```diff
@@ -166,7 +166,7 @@ String.prototype.escapeHTML = function(){
     return s.replace(/[\&"<>\\]/g, function(s) {
         switch(s) {
             case "&": return "&amp;";
-            case "\\": return "\\\\";
+            // case "\\": return "\\\\";
```

Comment:
@aleksandra-tarkowska Since you added this escapeHTML() function, you OK with removing this line from it?
I can't see that it's doing any good, but I might have missed something.